### PR TITLE
About page

### PR DESCRIPTION
**Pull request by pholls**
@justGrasse merge please?

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -18,12 +18,18 @@
 pat = User.create!( display_name: 'pat',
                     email: 'pat@email.com',
                     password: 'password' )
+jordan = User.create!(display_name: 'jordan',
```
Comment (justGrasse) [inline db/seeds.rb:21]: does it HAVE to be jordan???
meh.

File: db/seeds.rb [hunk 2]
```diff
@@ -18,12 +18,18 @@
...
+                    email: 'jordan@email.com',
+                    password: 'password' )
```
Comment (justGrasse) [inline db/seeds.rb:23]: doesn't he usually use 'stu'?

Comment (pholls) [reply db/seeds.rb:23]: Ugh how foolish of me

**Comment by justGrasse** [comment]
Looks great @pholls!  Thanks for getting that 'about' page in place!
My only concern is that interested folks are going to use up the demo games, and we'll need to re-seed to refresh their use. 
Here's a thought: we intercept ALL games being created as 'demo' and create a seeded game called 'demo788734823486'.  Just 'demo' w/a lil salt.

**Comment by pholls** [comment]
@justGrasse That's a good idea. Definitely not something I'm going to tackle tonight, but feel free to implement it if you'd like, and I'll review and merge your pull request. For now, I'm just going to seed the 50 demos and if I need to re-seed I'll do that.

**Pull request merged**